### ISSUE TITLE
Ukrainian sky culture

### ISSUE THREAD

**Issue by ghost** [Stellarium/stellarium#2997]
We have created 2 Ukrainian sky cultures. Link to traditional https://codeberg.org/Yurii/Stellarium_ukrainian_traditional_culture and to personal https://codeberg.org/Yurii/Stellarium_ukrainian_personal_culture.
Traditional based on traditional knowledge about constellations, personal is about newest Ukrainian history.

I want to make this sky cultures available to everyone using desktop or app.

**Comment by gzotti** [comment]
Thank you! The traditional may be accepted in the default distribution if you wish.  But please format the HTML of the description  following the other SCs. Is this really "ethnographic", not "traditional"? Also, the sources should not be just web addresses. I understand the current situation may be difficult, though. 

However, as much as I understand your motivation for your personal skyculture, Stellarium cannot be instrumentalized by including this one in the default distribution. I hope the link above will be enough so that users can find it.

**Comment by AnYurii** [comment]
Please explain about the sources. Maybe there is some best example? Because in those sky cultures, whose languages   I sampled, there were only references.

Traditional sky culture is really traditional, not ethnographic.

I have formatted description like in other cultures.

**Comment by AnYurii** [comment]
I made those changes. Do I need to change anything so that the traditional sky culture can be added to the Stellarium?

**Comment by gzotti** [comment]
Sorry for the delay. The easiest way for us is sending a "Pull Request". Instructions are in the wiki. @sushoff any comments on a new SC?

**Comment by sushoff** [comment]
I suggest to include only the traditional ethnographic one and not the personal one - because this is also the case for all other peoples. 

Issues: 

- [ ] your licence must be of the CC licence family! (GPL is for software, CC is for data)
- [ ] constellations.names:* "The Sehpherd" is a typo
* description* 
- [ ] in the first few sentences: please not only state your project but also your sources: from what time / historical epoch are these constellations considered to stem from and from where exactly (within today's Ukraine, there are several subcultures as far as I know from a supervised BA thesis ~2 years ago) 
- [ ] please let somebody else read the text for more typos (e.g. paranthesis missing after "M45") 
- [ ] thank you for the statement that constellations have nothing to do with mythology: this is the case in all traditional cultures (except the Greek one due to the simple fact that "the Greek" was an artificial construction with the political aim to merge many cultures in a newly built empire - so they invented stories to integrate foreign deities).

**Comment by AnYurii** [comment]
License was changed on the CC BY-SA

Fixed found mistakes

Added a few sentences about sources

Please let me know if it is possible to add a personal sky culture (New Ukrainian Sky) in user contributed sky cultures and how exactly to do it?
https://stellarium.org/skycultures-list.html

**Comment by sushoff** [comment]
Frankly spoken, I hesitate to vote for the distribution of this personal sky culture on the Stellarium official website. Due to international law, the naming of the sky (stars, constellations, mountains on planets of the solar system, landscapes on the Moon etc. ...) should be kept politically neutral. The global association of the IAU is the only institution with the authority to name celestial bodies. Therefore, I am personally against the creation of new constellations. Huge movements of the current culture like the Tolkin saga(s) or the Star Trek saga might possibly allow for exceptions. Still, as a historian of science I only want to map historical and indigenous constellations in the framework of my (and other people's) research on (1) transfer and transformation of knowledge, (2) creation of knowledge, (3) exchange and development of knownledge and ideas. Stellarium is therefore a research tool and the sky cultures are the fraction of (mathematical) research that can easily be shared in planetariums, museums and other outreach activities. From my point of view, there is no politics implied: neither for me nor for the IAU whose authority might be circumvent here. 

Feel free to create whatever artwork you like and feel free to publish it on your personal or institutional website(s) or even to link it in your traditional Ukrainian sky culture - but please keep the natural night sky and it's display in the free educational software Stellarium politically neutral and really accessible for everybody! 

With simultaneously expressing my empathy for everything you and your people have to suffer currently (as well as many other people at other places of the world who don't have the privilege to live in peace): **Thank for your understanding!**

**Comment by AnYurii** [comment]
Of course. Thanks for the explanation. I accept this position, I just did not know about it.

**Comment by gzotti** [comment]
OP has vanished and the SCs are no longer online. We can only close it here. I hope authors are well, the work is still available somewhere and interested people can find it.
